mesonbuild/interpreter.py

```diff
@@ -3733,6 +3733,13 @@ def func_add_test_setup(self, node, args, kwargs):
         timeout_multiplier = kwargs.get('timeout_multiplier', 1)
         if not isinstance(timeout_multiplier, int):
             raise InterpreterException('Timeout multiplier must be a number.')
+        is_default = kwargs.get('is_default', False)
+        if not isinstance(is_default, bool):
+            raise InterpreterException('Is_default option must be a boolean')
```

Comment:
Don't capitalize the kwarg name, I realize it's done that way elsewhere but it's confusing because users might think that kwargs are case-insensitive.